components/inputs/demo/input-group.html

```diff
@@ -26,20 +26,18 @@ <h2>Simple</h2>
 					<d2l-form>
 						<d2l-input-group>
 							<d2l-input-text label="Name" required></d2l-input-text>
-							<d2l-input-fieldset label="Options">
-								<d2l-input-checkbox-group>
-									<d2l-input-checkbox>Option 1</d2l-input-checkbox>
-									<d2l-input-checkbox checked>Option 2</d2l-input-checkbox>
-									<d2l-input-checkbox>Option 3</d2l-input-checkbox>
-								</d2l-input-checkbox-group>
-							</d2l-input-fieldset>
-							<d2l-input-fieldset label="Address" label-hidden>
+							<d2l-input-textarea label="Description" rows="3"></d2l-input-textarea>
+							<d2l-input-checkbox-group label="Options">
```

Comment:
This shouldn't have been using `<d2l-input-fieldset>` and instead should have been using checkbox group's label (which renders a fieldset).